packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+MetalQuartzLink.STATE = {
+	/**
+	 * Sets the data-onclick attribute on the anchor tag.
+	 * @instance
+	 * @memberof MetalQuartzLink
+	 * @type {?string}
+	 * default undefined
+	 */
+	dataOnclick: {
```

Comment:
@pat270 and I already discussed it, but we can remove this attribute since metal has a way of adding DOM event handlers to components via the `events` attribute.

```javascript
new MetalQuartsLink({
  events: {
    click: function(event) {
    }
  }
});
```